scripts/test.sh

```diff
@@ -19,5 +19,5 @@ if [ -n "$BUILD_ONLY" ]; then
   fi
   echo "Skipping test run because of BUILD_ONLY environment variable"
 else
-  CTEST_OUTPUT_ON_FAILURE=1 CTEST_PARALLEL_LEVEL=5 make -j6 check-full
+  CTEST_OUTPUT_ON_FAILURE=1 CTEST_PARALLEL_LEVEL=4 make -j6 check-full
```

Comment:
Just curious, have you experienced differing performance with different parallel levels here?

Reply:
I did this because worker was failnig bcause after changes the order of tests has changed and ctest runs 5 very long running tests at once. As result  there is no output for some long time and travis kills the worker.